### PR TITLE
Do not show quarterly release part of the version in UI.

### PR DESCRIPTION
**Pull request by rvykydal**
Resolves: rhbz#1671384

### CODE REVIEW

File: pyanaconda/product.py [hunk 1]
```diff
@@ -50,6 +50,10 @@
     productArch = productStamp[productStamp.index(".") + 1:]      # pylint: disable=no-member
 if productVersion == "development":
     productVersion = "rawhide"
+# 8.0.X -> 8.0
+if productVersion.count('.') == 2:
```
Comment (poncovka) [inline pyanaconda/product.py]: Maybe it could be `>= 2`. Just in case. The remaining code should still work fine.

Comment (jkonecny12) [reply pyanaconda/product.py]: Good point.

Comment (rvykydal) [reply pyanaconda/product.py]: Agreed, updated

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
PR with port to Fedora: https://github.com/rhinstaller/anaconda/pull/2400

**Comment by rvykydal** [comment]
Ported to master with https://github.com/rhinstaller/anaconda/pull/2400

**Pull request merged**